File: java/FizzBuzz.java
class Main {
  public static void main(String[] args) {
    int iterations = 100;
    String say;
    for(int i=1; i<=iterations; i++){
      say = "";
      if(i%3==0){
        say += "Fizz";
      }
      if(i%5==0){
        say += "Buzz";
      }
      if(say.equals("")){
        System.out.println(i);
      }
      else{
        System.out.println(say);
      }
    }
  }
}

File: ruby/FizzBuzz.rb
iterations = 100

for i in 1..iterations
  say = ""
  if i%3==0
    say += "Fizz"
  end
  if i%5==0
    say += "Buzz"
  end
  if say == ""
    puts i 
  else
    puts say
  end
end 